stable/elasticsearch/templates/configmap.yaml

```diff
@@ -65,17 +67,37 @@ data:
         ping.unicast.hosts: ${DISCOVERY_SERVICE:}
         minimum_master_nodes: ${MINIMUM_MASTER_NODES:2}
 
-{{- if and ( .Values.cluster.xpackEnable ) ( gt $minorAppVersion 6.3 )  }}
+  {{- if and ( .Values.cluster.xpackEnable ) ( gt $minorAppVersion 6.3 )  }}
     # see https://www.elastic.co/guide/en/x-pack/current/xpack-settings.html
     # After 6.3 xpack systems changed and are enabled by default and different configs manage them this enables monitoring
     xpack.monitoring.collection.enabled: ${XPACK_MONITORING_ENABLED:false}
-{{- else if .Values.cluster.xpackEnable }}
+  {{- else if .Values.cluster.xpackEnable }}
     # see https://www.elastic.co/guide/en/x-pack/current/xpack-settings.html
     xpack.ml.enabled: ${XPACK_ML_ENABLED:false}
     xpack.monitoring.enabled: ${XPACK_MONITORING_ENABLED:false}
     xpack.security.enabled: ${XPACK_SECURITY_ENABLED:false}
     xpack.watcher.enabled: ${XPACK_WATCHER_ENABLED:false}
-{{- end }}
+  {{- end }}
+{{- else if hasPrefix "7." .Values.appVersion }}
+    # see https://github.com/kubernetes/kubernetes/issues/3595
```

Comment:
Is this specific to ES7?

Reply:
Honestly, I have no idea about this issue and es7, but es nodes fail with `bootstrap.memory_lock: true` option.

Reply:
We should check this and find the source before blindly merge it.

Reply:
It seems that [6.7 doc](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/_memory_lock_check.html) and [7.0 doc](https://www.elastic.co/guide/en/elasticsearch/reference/7.0/_memory_lock_check.html) are same and nothing has changed about this issue.

and reference in elastic's own chart https://github.com/elastic/helm-charts/issues/7